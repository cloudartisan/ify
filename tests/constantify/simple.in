All
of
these
should
become
constant
expressions
