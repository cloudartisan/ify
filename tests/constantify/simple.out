ALL = 'All'
OF = 'of'
THESE = 'these'
SHOULD = 'should'
BECOME = 'become'
CONSTANT = 'constant'
EXPRESSIONS = 'expressions'
